Jenkinsfile_CNP

```diff
@@ -71,27 +71,27 @@ withPipeline("nodejs", product, component) {
 
   //if (env.CHANGE_TITLE && !env.CHANGE_TITLE.startsWith('[PREVIEW]')) {
     enableDockerBuild()
-    enableDeployToAKS()
+    installCharts()
 //  }
   setVaultName('probate')
   loadVaultSecrets(secrets)
   enableSlackNotifications('#probate-jenkins')
-  after('aksdeploy') {
-    def subscription = env.SUBSCRIPTION_NAME
-    def aksServiceName = "${product}-${component}-${env.BRANCH_NAME}".toLowerCase()
-    def aksDomain = "${(env.SUBSCRIPTION_NAME in ['nonprod', 'prod']) ? 'service.core-compute-preview.internal' : 'service.core-compute-saat.internal'}"
-
-    def kubectl = new Kubectl(this, subscription, aksServiceName)
-
-    def ingressIP = kubectl.getServiceLoadbalancerIP("traefik", "kube-system")
-
-    def consul = new Consul(this)
-
-    consul.registerDns("feature-toggle-api-${aksServiceName}", ingressIP)
-
-    env.FEATURE_TOGGLES_API_URL="https://feature-toggle-api-${aksServiceName}.${aksDomain}"
-    echo "${env.FEATURE_TOGGLES_API_URL}"
-  }
+//  after('aksdeploy') {
```

Comment:
any reason you didn't remove all this?

Reply:
Good point, was left around trying to get api ingress working. I'll remove it once Ive had a regular PR go through OK.